plugin/builtin/patchstrategicmergetransformer/PatchStrategicMergeTransformer_test.go

```diff
@@ -319,7 +340,25 @@ spec:
         name: nginx
       - image: busybox
         name: busybox
-`)
+`, `
+apiVersion: apps/v1
```

Comment:
Is this expected value correct? Looks that the patch2 is not applied.

Reply:
It's correct in that it's what happens.  It's incorrect in that it needs to be fixed.
This PR is about defining what remains to be fixed.
I needed an upper bound on the work.
All remaining work is covered by #3304, and i want to get it done this week.